ninja: error: '../../third_party/typescript/linux-amd64/src/lib/tsc', needed by 'gen/chrome/browser/resources/extensions_zero_state_promo/build_ts_manifest.json', missing and 

--- a/third_party/devtools-frontend/src/third_party/typescript/typescript.py
+++ b/third_party/devtools-frontend/src/third_party/typescript/typescript.py
@@ -10,6 +10,7 @@ import os
 
 
 def GetBinaryPath():
+    return '/usr/bin/tsc'
     if platform.machine() == 'arm64':
         darwin_path = 'mac-arm64'
     else:
--- a/third_party/typescript/tsgo.gni
+++ b/third_party/typescript/tsgo.gni
@@ -23,7 +23,6 @@ assert(defined(_platform_dir), "Did not
 
 tsgo_inputs = [
   "//third_party/typescript/typescript.py",
-  "//third_party/typescript/$_platform_dir/src/lib/$_binary_name",
 ]
 
 _tsgo_lib_files = [
@@ -137,6 +136,3 @@ _tsgo_lib_files = [
   "lib.webworker.iterable.d.ts",
 ]
 
-foreach(file, _tsgo_lib_files) {
-  tsgo_inputs += [ "//third_party/typescript/$_platform_dir/src/lib/$file" ]
-}
